.github/workflows/release.yml

```diff
@@ -221,15 +183,42 @@ jobs:
 
           # 2. Copy and fix dependencies
           destDir=$APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
-          process_dependencies $target $destDir $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres "@executable_path/../Resources/lib"
-
-          # 3. Copy loaders
           mkdir -p $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders
-          cp -r /opt/homebrew/lib/gdk-pixbuf-2.0/2.10.0/loaders/*.so $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders
 
-          # 4. Fix loaders
+          libs="cairo fontconfig freetype fribidi gdk-pixbuf gettext glib graphene graphite2 gtk4 harfbuzz jpeg-turbo libepoxy libpng libtiff librsvg libx11 libxau libxcb libxdmcp libxext libxrender lzo pango pcre2 pixman xz zstd"
+          for lib in $libs; do
```

Comment:
This seems fragile as libraries change, why not copy all the libraries that were installed, can't something like this work?:
```
cp /opt/homebrew/opt/*/lib/*.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
```

Reply:
@nazar-pc that works for sure! but that also copy other libs that the app don't need (as we're using brew to install libtool, automake. And maybe more in future). What do you think?

Reply:
Hm... that is really annoying. Let's keep it as is then. I don't want to get application unexpectedly broken once GTK updates dependencies.

If you can send a PR with fixed version that'd be great, but otherwise I think this is all, thanks!

Reply:
Unfortunately brew don't have an option to install a specific version of library. May be at some point in the future, we can create an archive of dependencies and just download it instead of depends on brew binaries? That way we can always guarantee the lib version won't change between each release.

Reply:
I would really prefer not to maintain that, installing latest version every time package is built is much easier. Your initial approach was better, let's keep it then.